internal/resources/policy/kind/mutation/constant.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package policykindmutation
+
+import (
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy/kind/mutation/recipe"
+)
+
+const (
```

Comment:
Can we define all policy-related constants in `/policy/common/constants.go` file? Having multiple files can be confusing. Better to have one file and name the constants per policy, like `MutationPodSecurityRecipe`, `MutationLabelRecipe` etc.

Reply:
I would say currently used policy constant usage approach is more suitable. This common `/policy/common/constants.go` should be for values that could be used for multiple policy recipes. Constant values can be seen as implementation details that should not leak out to other packages/directories if there is no reason/usage.